packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -27,9 +27,11 @@ class BottomNavigationBarItem {
   const BottomNavigationBarItem({
     @required this.icon,
     this.title,
```

Comment:
I think we should add the deprecation notice in the constructor too, [see example](https://github.com/flutter/flutter/blob/5cfb16b193886e68e51cc7256389d78a21f31bfd/packages/flutter/lib/src/material/text_theme.dart#L129).

Also it mention after what version it was deprecated (https://github.com/flutter/flutter/wiki/Tree-hygiene#deprecation) and you might want to add a note that this is to improve text scaling support in the deprecation notice.

Reply:
Done.